spec/system/placements/schools/placements/view_a_placement_spec.rb

```diff
@@ -0,0 +1,276 @@
+require "rails_helper"
+
+RSpec.describe "Placements / Schools / Placements / View a placement",
+               type: :system, service: :placements do
+  let!(:school) { create(:placements_school, name: "School 1", phase: "Primary") }
+  let!(:placement) { create(:placement, school:) }
+  let!(:subject_1) { create(:subject, name: "Subject 1", subject_area: "primary") }
+
+  before do
+    given_i_sign_in_as_anne
+  end
+
+  context "with subjects" do
+    let!(:subject_2) { create(:subject, name: "Subject 2") }
+    let!(:subject_3) { create(:subject, name: "Subject 3") }
+
+    scenario "User views a placement with one subject" do
+      given_a_placement_has_one_subject(subject_1)
+      when_i_visit_the_placement_show_page
+      then_i_see_the_subject_name_in_the_placement_details("Subject 1")
+    end
+
+    scenario "User views a placement with multiple subjects" do
+      given_a_placement_with_multiple_subjects([subject_1, subject_2, subject_3])
+      when_i_visit_the_placement_show_page
+      then_i_see_all_of_the_subjects_names_in_the_placement_details(
+        ["Subject 1", "Subject 2", "Subject 3"],
+      )
+    end
+  end
+
+  context "with mentors" do
+    let!(:mentor_1) { create(:placements_mentor, first_name: "Joe", last_name: "Bloggs") }
+    let!(:mentor_2) { create(:placements_mentor, first_name: "John", last_name: "Doe") }
+    let!(:mentor_3) { create(:placements_mentor, first_name: "Agatha", last_name: "Christie") }
+
+    before do
+      given_a_placement_has_one_subject(subject_1)
+      given_the_school_has_mentors(school:, mentors: [mentor_1, mentor_2, mentor_3])
+    end
+
+    scenario "User views a placement with one mentor" do
+      given_a_placement_with_one_mentor(mentor_1)
+      when_i_visit_the_placement_show_page
+      then_i_see_the_mentor_name_in_the_placement_details("Joe Bloggs")
+    end
+
+    scenario "User views a placement with multiple mentors" do
+      given_a_placement_with_multiple_mentors([mentor_1, mentor_2, mentor_3])
+      when_i_visit_the_placement_show_page
+      then_i_see_all_of_the_mentors_names_in_the_placement_details(
+        ["Joe Bloggs", "John Doe", "Agatha Christie"],
+      )
+    end
+  end
+
+  context "without a mentors" do
```

Comment:
```suggestion
  context "without a mentor" do
```

Reply:
resolved in [5a5c21a](https://github.com/DFE-Digital/itt-mentor-services/pull/325/commits/5a5c21a937ac7f819a6ad8309c14c9c61b31c749)